.github/workflows/main.yml

```diff
@@ -224,6 +230,9 @@ jobs:
           cd ENHSP-Public; git checkout enhsp20-0.9.5; bash ./compile; cd ..
           bash -c "mkdir .planners; mv ENHSP-Public/enhsp-dist .planners/enhsp-20; rm -rf ENHSP-Public"
 
+      - name: Download and install FMAP
+        run: python3 -m pip install up-fmap
+
```

Comment:
The changes in this file are not needed since FMAP is not used in the tests.

Reply:
Ok, I've removed this for now! Do you suggest adding a test for FMAP?